Index: src/main/java/org/travel/Core/Main.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/main/java/org/travel/Core/Main.java b/src/main/java/org/travel/Tests/Main.java
rename from src/main/java/org/travel/Core/Main.java
rename to src/main/java/org/travel/Tests/Main.java
--- a/src/main/java/org/travel/Core/Main.java	(revision 69284e2a9e365468f4e79e75014dabf6ba89f69a)
+++ b/src/main/java/org/travel/Tests/Main.java	(date 1762249746251)
@@ -1,4 +1,4 @@
-package main.java.org.travel.Core;
+package main.java.org.travel.Tests;
 
 import main.java.org.travel.Core.Domain.Route;
 import main.java.org.travel.Core.Service.RouteService;
Index: src/main/java/org/travel/DatabaseConnectionTester.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/main/java/org/travel/DatabaseConnectionTester.java b/src/main/java/org/travel/Tests/DatabaseConnectionTester.java
rename from src/main/java/org/travel/DatabaseConnectionTester.java
rename to src/main/java/org/travel/Tests/DatabaseConnectionTester.java
--- a/src/main/java/org/travel/DatabaseConnectionTester.java	(revision 69284e2a9e365468f4e79e75014dabf6ba89f69a)
+++ b/src/main/java/org/travel/Tests/DatabaseConnectionTester.java	(date 1762249746277)
@@ -1,4 +1,4 @@
-package main.java.org.travel;
+package main.java.org.travel.Tests;
 
 import com.mongodb.client.MongoDatabase;
 import main.java.org.travel.Adapters.TripDataMongoAdapter;
